src/tools/histogram_dump.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# coding: utf-8
+#
+# Ceph - scalable distributed file system
+#
+# Copyright (C) 2017 OVH
+#
+# This is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public
+# License version 2, as published by the Free Software
+# Foundation.  See file COPYING.
+#
+
+import json
+import subprocess
+import time
+import os
+import argparse
+
+
+def shorten(val):
+    if type(val) is str:
```

Comment:
True, converted to ``instanceof`` (although this is a bit tricky to get right and compatible with both python2 and python3 at the same time)